config/deploy/production.rb

```diff
@@ -1,61 +1,11 @@
 # server-based syntax
 # ======================
-# Defines a single server with a list of roles and multiple properties.
-# You can define all roles on a single server, or split them:
-
-# server 'example.com', user: 'deploy', roles: %w{app db web}, my_property: :my_value
-# server 'example.com', user: 'deploy', roles: %w{app web}, other_property: :other_value
-# server 'db.example.com', user: 'deploy', roles: %w{db}
-server 'signage.chapman.edu', user: 'charles', roles: %w{app db web}
-
+server 'signage.chapman.edu', user: 'charles', roles: %w{app db web postgres}
```

Comment:
It really doesn't matter. We can also use the `app`, `web`, or `db` role. I did it for consistency with the Inside app, where it was done because we have our database on a different server.